infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -245,15 +246,17 @@ public TokenType txnCanCommit(TxResolutionInfo txInfo, /** Input. */
                     if (v != null && v > txSnapshotTimestamp) {
                         log.debug("ABORT[{}] conflict-key[{}](ts={})", txInfo, conflictParam, v);
```

Comment:
I think this too can be an info or warm. I do not think we wan't abort's to be very high in general. It might be good to log this when it starts happening.